src/test/java/io/cdap/plugin/gcp/dataplex/source/DataplexInputFormatProviderTest.java

```diff
@@ -0,0 +1,105 @@
+package io.cdap.plugin.gcp.dataplex.source;
+
+import com.google.auth.oauth2.GoogleCredentials;
+import com.google.cloud.dataplex.v1.StorageSystem;
+import io.cdap.plugin.common.batch.ConfigurationUtils;
+import io.cdap.plugin.gcp.common.GCPUtils;
+import io.cdap.plugin.gcp.dataplex.common.util.DataplexConstants;
+import io.cdap.plugin.gcp.dataplex.common.util.DataplexUtil;
+
+import org.apache.hadoop.conf.Configuration;
+import org.apache.hadoop.mapreduce.InputFormat;
+import org.apache.hadoop.mapreduce.InputSplit;
+import org.apache.hadoop.mapreduce.JobContext;
+import org.apache.hadoop.mapreduce.TaskAttemptContext;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.powermock.api.mockito.PowerMockito;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Spliterator;
+import java.util.stream.StreamSupport;
+
+import static org.powermock.api.mockito.PowerMockito.mock;
+import static org.powermock.api.mockito.PowerMockito.when;
+
+@RunWith(PowerMockRunner.class)
+@PrepareForTest({ConfigurationUtils.class, GCPUtils.class, GoogleCredentials.class, DataplexUtil.class})
+public class DataplexInputFormatProviderTest {
+
+  @Mock
+  JobContext mockContext;
+
+  @Mock
+  private Configuration configuration;
```

Comment:
A mock object for the `Configuration` class has been created at the class level but is not being used. Instead, the mock object is created in every test case at the method level. If we aren't setting new values to it in every test case, we can skip the creation at the method level and just declare it once at the class level and use it everywhere.

Reply:
done